src/exporter/DaemonMetricCollector.h

```diff
@@ -15,8 +15,9 @@
 class DaemonMetricCollector {
 public:
   void main();
+  void set_sock_dir();
   std::string get_metrics();
-  static const char *SOCKETDIR;
+  std::string SOCKETDIR = "/var/run/ceph/";
```

Comment:
If we have already defined a default in the config options, we shouldn't define it again.